docs/cmdline-opts/resolve.d

```diff
@@ -11,6 +11,10 @@ the number used for the specific protocol the host will be used for. It means
 you need several entries if you want to provide address for the same host but
 different ports.
 
+By specifying '*' as host you can tell curl to resolve any host and specific
+port pair to the specified address. Wildcard is resolved last so any --resolve
+with a specific host and port will be used first.
```

Comment:
I think it would be good to mention the version number where the wildcard support is added.

Reply:
Good point, done.